definitions/checks/foreman_tasks/not_running.rb

```diff
@@ -9,8 +9,17 @@ class NotRunning < ForemanMaintain::Check
     end
 
     def run
-      assert(feature(:foreman_tasks).running_tasks_count == 0,
-             'There are actively running tasks in the system')
+      task_count = feature(:foreman_tasks).running_tasks_count
+      assert(task_count == 0,
+             failure_message(task_count),
+             :next_steps =>
+               [Procedures::ForemanTasks::Resume.new,
+                Procedures::ForemanTasks::UiInvestigate.new('search_query' => 'state = paused')])
```

Comment:
The search query should be `state = running`, shouln't it?

Reply:
Yes, actually!!